### PR TITLE
DOC: Change in the documentation for chebpts2 method

### PR DESCRIPTION
**Pull request by shashank140195**
fix for the issue chebpts2 Docstring #22274

### CODE REVIEW

File: numpy/polynomial/chebyshev.py [hunk 1]
```diff
@@ -1959,7 +1959,7 @@ def chebpts2(npts):
     Chebyshev points of the second kind.
 
     The Chebyshev points of the second kind are the points ``cos(x)``,
-    where ``x = [pi*k/(npts - 1) for k in range(npts)]``.
+    where ``x = [pi*k/(npts - 1) for k in range(-npts+1,1)]``.
```
Comment (charris) [inline numpy/polynomial/chebyshev.py:1962]: Same points, different order. Hmm, I suppose it doesn't hurt.

**Comment by charris** [comment]
I'm not convinced this is an improvement, it is just the same points in different order. I left some comments on issue #22274.

**Comment by Pro-Coder04** [comment]
> I'm not convinced this is an improvement, it is just the same points in different order. I left some comments on issue #22274.

What changes are you expecting in here?

**Comment by seberg** [comment]
@charris can you make (any) call on what you prefer here?  I guess the change would ensure that the text reproduces the order of the result?  (Adding a `[::-1]` might also a be a way to achieve that.)

**Comment by charris** [comment]
I would keep the current math as it is easier to understand, at least for me, but a comment "sorted in ascending order" might help. Your suggestion is also a good one and maybe better for simplicity. Let's see if the authors have any preference.

**Review by rossbar — COMMENTED** [review]
I took the liberty of applying @charris' suggestion just to help move this along. I'll let @seberg or @charris make the final call on whether this looks fine.

Note it'd probably be good to squash-merge this one (I didn't to force push).

**Comment by charris** [comment]
Thanks @shashank140195 and @rossbar.

**Comment by shashank140195** [comment]
Yeah I believe Charles suggestion "sorted in ascending order" would be the
best option to avoid any confusion.

**Pull request merged**